.changelog/unreleased/improvements/2522-privval-skip-extension-signing.md

```diff
@@ -0,0 +1,2 @@
+- `[proto]` Add `skip_extension_signing` field to the `SignVoteRequest` message
```

Comment:
Can you rename the field in the original changelog entry as well?

Reply:
Can't we replace or amend the existing one?

Reply:
That's an option, not sure why another one was added.

Reply:
Oh sorry, I failed to notice the entry from #2496. Will change that instead.

Reply:
Updated in https://github.com/cometbft/cometbft/pull/2522/commits/9eba77f0ec9c9807c93b572d79077a3c1034dabe
I did not remove my entry because it details changes in `proto` that the entry for #2496 did not mention.